io.github.Bollos00.LibreMines.json

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "io.github.Bollos00.LibreMines",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "libremines",
+    "rename-desktop-file": "libremines.desktop",
+    "rename-icon": "libremines",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
Why do you need host filesystem access?

Reply:
I followed the [example of the Qt application on the documentation](https://docs.flatpak.org/en/latest/qt.html).

I can test building the package without that line if you prefer.

Reply:
See Item 3 in the check list you "checked"

Reply:
At first glance, nothing in the app seems to require the host access. Please remove it.